docs/shapes.md

```diff
@@ -1,11 +1,36 @@
 # Shapes and layout
 
+## Structure of an XLA Op
+
+Consider an example Op:
+
+```
+%fusion.3 = bf16[32,32,4096]{2,1,0:T(8,128)(2,1)S(1)} fusion(bf16[32,32,8192]{2,1,0:T(8,128)(2,1)S(1)} %fusion.32), kind=kCustom, calls=%all-reduce-scatter.3
+```
+
+This consists on the following main components:
+
+* Op Name: `fusion.3`
+  * A dot or fusion op is a set of operations containing at most 1 matrix multiplication and possibly a bunch of related pointwise VPU-ops.
```

Comment:
Why focus on dot of fusion ops, this is a general doc for all ops, right? Also this sentence isn't quite right. A dot is a standalone op and not a set of operations, a fusion is also a standalone op that happens to point to  a computation. That computation is a set of ops.

Could you please clarify what you wanted to explain here?

Reply:
Thanks for pointing out, this is an oversight from initially coping the example from the JAX scaling book. I've still added a brief explanation of fusion (mostly to not leave it as jargon, and because I didn't see it described anywhere else in the docs), but de-emphasized. Instead, this description now focuses more on naming.